gatsby-browser.js

```diff
@@ -177,14 +177,10 @@ export const onClientEntry = () => {
 }
 
 const scrollToAnchor = (location) => {
-    // Check for location so build does not fail
-    if (location && location.hash) {
-        window.scrollTo({
-            top: window.innerHeight,
-            behavior: 'smooth',
-        })
+    if (location.hash) {
+        const elem = document.querySelector(location.hash)
+        elem?.scrollIntoView({ behavior: 'smooth' }) || window.scrollTo(0, 0)
     }
-
     return true
 }
 
```

Comment:
@hasan-deriv as we are applying this on top level it will change behaviour on other pages , can you go to market page `derived` and then change the tab from option to CFD or any other tab and compare to prod  we are scrolling to top which correct behaviour should stay at same position and change the tab only 🙏

Reply:
@mitra-deriv, I removed the code from gatsby-browser.js. and only fix it for helo-center. So, it won't affect other pages.